detox/src/errors/DetoxConfigErrorBuilder.js

```diff
@@ -87,6 +87,15 @@ class DetoxConfigErrorBuilder {
     });
   }
 
+  malformedAppLaunchArgs() {
```

Comment:
What about unit tests here?

Reply:
Ooooh, right. 😄 I've really forgotten I had restored snapshot tests there... 😊 

and... pssst, man...

![](https://lparchive.org/Legend-of-Kyrandia-3-Malcolms-Revenge/Update%2006/4-kyrandia3_update4_003.png)

Just make sure to not tell Leo about those snapshot tests. 😆

Reply:
hehe that is probably the weirdest reference I've ever come across